.github/workflows/ci.yaml

```diff
@@ -8,6 +8,11 @@ concurrency:
 
 on:
   pull_request:
+    paths-ignore:
```

Comment:
We won't be merging this PR anytime soon, probably ever. Because Kafka docs are published through Discourse now. Can we have the same CI changes to the repo in a dedicated PR please?